lua/lualine/components/filename.lua

```diff
@@ -1,6 +1,31 @@
-local function filename()
-  local data = [[%t %m]]
-  return data
+local function filename(args)
+  -- setting defaults
+  local modified, full_name, relative = true, false, true
```

Comment:
`full_name` and `relative` don't describe well what's happening here. My first thought was that `full_name` is the same as `full_path`, which is exactly opposite of what `relative` means.Should probably be named `shorten` and `full_path` respectively.

Reply:
ok

Reply:
renamed modified to file_status, full_name to full_path

Reply:
I looked at this 2 days ago and I already don't know what does what. Please come up with better naming scheme.

Reply:
what's better than full_path & relative ?

Reply:
Ok my bad . At first I thought you ment
full_name -> full_path & relative -> shorten.

But now I realize you ment fullname -> shorten & relative -> full_path.

This does make sence I've renamed them accordingly.

Reply:
yep, that's exactly what I meant, thanks